app/Repositories/DashboardRepository.php

```diff
@@ -44,16 +46,19 @@ public function getWorkReport($input)
 
         $data = [];
         $totalRecords = 0;
+        $index=0;
```

Comment:
code must be formatted 
```suggestion
        $index = 0;
```